packages/lesswrong/lib/faviconBadge.ts

```diff
@@ -0,0 +1,12 @@
+import { faviconUrlSetting, faviconWithBadgeSetting } from "../components/common/HeadTags";
```

Comment:
I assume this file wasn't intended to be in this PR - seems ok to leave it in if the other one's going to be landing soon, just flagging

Reply:
Not meant to be in this PR, and doesn't typecheck outside of the PR it's supposed to be in, so, removed.

(Habit-bug: `git stash` by default doesn't stash unrecognized files, but can be manually directed to stash them.)